### DISCUSSION TITLE
Images use different path in tables between markdown preview and html

### DISCUSSION THREAD

**Discussion by stephan1827** [mkdocs/mkdocs#2865] [Q&A] [answered]
I tried to create a table without header. The image shows up correct without a table in the markdown preview and when deployed when I use the following code

`![](./images/import1.png)`

But when I copy and paste that code into a table it will shows up correct in the markdown preview but not when deployed.  When I use this path with 2 dots at the beginning

`![](../images/import1.png)`

it will show up correct when deployed but not in markdown preview. 

Here is a table with both versions in one table. This is how it looks in vscode

![image](https://user-images.githubusercontent.com/2972703/172838635-a840e0ff-3292-4baa-992a-fe43d5a469a1.png)

and this is how it looks when deployed

![image](https://user-images.githubusercontent.com/2972703/172839669-053c3eef-c550-49f1-9499-e946b5b16d6e.png)

Is there any way to set the path so that it shows up correct in preview and when deplyed?

**Comment by stephan1827** [comment, marked as answer]
It seems that the problem is caused by the fact that mkdocs creates directories for each page. Using the parmeter 

use_directory_urls: false 

fixes the issue

**Comment by bmfs** [comment]
Actually this still is an issue/bug.
Image url rendering is different when inside tables.

**Reply by kamilkrzyskow** [reply]
@bmfs 
If you're using HTML tags inside the tables then this is not a bug. Markdown paths refer to the filesystem from the source. 

These path are processed by MkDocs / Markdown processor and are properly converted based on the `use_directory_urls` setting, which is `true` by default, since the nice names URLs end with a slash `/` e.g. `nice-name/`, the generated HTML paths need to use `../` to go back up one level.

If you're using `<tag src="path">` this path isn't processed and needs to be input manually to match the site settings.
